format/FormatPYunspecified.py

```diff
@@ -176,37 +174,6 @@ def _scan(self):
                 epochs={1: self._timesec},
             )
 
-    def get_mask(self, goniometer=None):
```

Comment:
This code was written by Tara Michaels-Clark in 2015. It's not clear exactly to me what it is supposed to do but it looks like the first section applies an untrusted region mask and then the second bit applies a trusted range mask. 

In the new C++ code, the trusted range mask will be set so that is fine; however, all the stuff between line 180 and line 200 will not be applied. @phyy-nx would it be possible to apply the tile stuff as untrusted rectangles in the panel model?